crates/net/downloaders/src/test_utils/file_client.rs

```diff
@@ -116,7 +224,7 @@ impl BodiesClient for FileClient {
 
 impl DownloadClient for FileClient {
     fn report_bad_message(&self, _peer_id: PeerId) {
-        warn!("Reported a bad message on a file client, the file may be corrupted or invalid");
+        panic!("Reported a bad message on a file client, the file may be corrupted or invalid");
```

Comment:
Yeah, this path is just a no-op. I don't remember exactly how invalid blocks are meant to be handled, I think we just stop sync at that point(?). I would check what geth does here since the hive tests are based on geth